reviewdog.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/haya14busa/reviewdog/diff"
 )
 
+var unused = 1
```

Comment:
**[unused]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var unused is unused (U1000)